<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Real Home Location</string>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbRHLatitude">
        <property name="text">
         <string>35.2576900</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbRHLongitude">
        <property name="text">
         <string>-97.602600</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>87</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pbRHGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="tbDisplay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>601</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Temp Home Location</string>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbTHLatitude">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbTHLongitude">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>87</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="pbtHGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Delay</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="lstMoves">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>261</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbProcessMoves">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNW">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbW">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbE">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSW">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>127</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ephemeris file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbEphemeris">
       <property name="text">
        <string>brdc2960.22n</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>127</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Duration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbDefualtDuration">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Last Location</string>
    </property>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>16</width>
       <height>61</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Latitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbLastLatitude">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>127</width>
       <height>45</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Longitude</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tbLastLongitude">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>87</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QPushButton" name="pbLastLocateGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbRecenter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>ReCenter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRHGenerate_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTravelTime">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>127</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Dist from TH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tbDistTempHome"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
